database/script.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE php_mysql_crud_valeria;
+
+USE php_mysql_crud_valeria;
+
+CREATE TABLE task(
+    id INT(11) PRIMARY KEY AUTO_INCREMENT,
```

Comment:
I think that you can do best this part of you code

you can do this

`id INT(15) PRIMARY KEY AUTO_INCREMENT`